app/views/users/show.html.erb

```diff
@@ -5,6 +5,10 @@
 
 <%= content_for :page_meta do %>
   <%= render "users/meta" %>
+  <% if @user.score < 0 || @user.suspended? %>
```

Comment:
In other parts of the app the `Style/NumericPredicate` cop would flag this but I guess it doesn't really check views the same way. So to be consistent we should change it like this:

```suggestion
  <% if @user.score.negative? || @user.suspended? %>
```